presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -554,6 +554,46 @@ Drop a schema::
 
     DROP SCHEMA iceberg.web
 
+Register table
+^^^^^^^^^^^^
+
+Iceberg tables for which table data and metadata already exist in the
+file system can be registered with the catalog using the ``register_table``
+procedure on the catalog's ``system`` schema by supplying the target schema,
+desired table name, and the location of the table data and metadata::
+
+    CALL iceberg.system.register_table("schema_name", "table_name", "hdfs://localhost:9000/path/to/iceberg/table/dir")
```

Comment:
I think we should use `'` rather than `"` for parameters are constan.

Reply:
Great point!